projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-list/query-list.component.ts

```diff
@@ -25,50 +26,36 @@ export class QueryListComponent implements OnChanges {
     }
   }
 
-  public sortTable(col: queryListColumn) {
+  public sortTable(col: QueryListColumn) {
```

Comment:
@DavidJayakumar - please add a return type for this function (even if `void`)

Reply:
as per Dan's comment